CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+
+## Running tests from the command line
+
+To run tests, you have to configure the download token for the Mapbox described [here][3] and then set environment variables:
+
+- `ABLY_API_KEY` - you can find this key in **your account -> {app name} -> API keys** on https://ably.com
+- `MAPBOX_ACCESS_TOKEN` - you can find this key in **your account -> tokens** on https://mapbox.com
+
+`ABLY_API_KEY` and `MAPBOX_ACCESS_TOKEN` are required to generate the `Secrets.swift` file.
+
+Run `Scripts/test.sh` to start the tests.
+
+## Running tests from Xcode IDE
+
+The recommended IDE for working on this project is the [Xcode][4].
+
+To open the project in the Xcode IDE, double click on the `Package.swift` file.
+
+To run tests from Xcode IDE, select the `ably-asset-tracking-swift-Package` scheme, select **_Product_** **_\-> Test_** _or use the keyboard shortcut_ **⌘U**
+
+## Building Platform-Specific Documentation
+
+_This repo uses_ [_jazzy_][5] _to build documentation._
+
+Run `bundle install` to install the required tools.
+
+Run `jazzy/build.sh` to build the documentation.
+
+The above command will generate HTML files that are located in the `docs` directory.
+
+## Release Process
+
+Releases should always be made through a release pull request (PR), which must bump the version number and add to the [changelog][6].
+
+The release process must include the following steps:
+
+1.  Ensure that all work intended for this release has landed on to `main` branch
+2.  Create a release branch named like release/1.2.3
+3.  Add a commit to bump the version number
+4.  Add a commit to update the changelog
+5.  Push the release branch to GitHub
+6.  Open a PR for the release against the release branch you just pushed
+7.  Gain approval(s) for the release PR from maintainer(s)
+8.  Land the release PR to the `main` branch
+9.  Create a tag named like v1.2.3 and push it to GitHub
+
+## Coding Conventions and Style Guide
+- The SDKs are written in Swift, however they still have to be compatible for use from Objective-C based apps.
+- Favor Protocol Oriented Programming with Dependency Injection when writing any code. We're unable to create automatic mocks in Swift, so it'll be helpful for writing unit tests.
+- SwiftLint is integrated into the project. Make sure that your code does not add any SwiftLint related warning.
+- Please remove default Xcode header comments (with author, license and creation date) as they're not necessary.
+- If you're adding or modifying any part of the public interface of SDK, please also update [QuickHelp](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/SymbolDocumentation.html#//apple_ref/doc/uid/TP40016497-CH51-SW1) documentation.
+- Docs are written for both Swift and ObjC
+
+[1]: https://github.com/apple/swift-package-manager
```

Comment:
Why are these here? I've commented against the same in the main readme.

Reply:
fixed here a0f4158353273444802905e244779111e9d21a3c